src/Symfony/Component/Form/FormInterface.php

```diff
@@ -260,6 +260,11 @@ public function initialize(): static;
      */
     public function handleRequest(mixed $request = null): static;
 
+    /**
+     * Call {@link handleRequest()} and return the result of the condition {@link isSubmitted()} and {@link isValid()}
+     */
+    public function isSubmittedRequestValid(mixed $request = null): bool;
+
```

Comment:
Why are we still changing the interface, now that you've moved this functionality to `AbstractController`?